easybuild/easyconfigs/c/CubeW/CubeW-4.4-CrayGNU-18.07.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+name = 'CubeW'
+version = '4.4'
+
+homepage = 'http://www.scalasca.org/software/cube-4.x/download.html'
+description = """Cube Writer Library is a high-performance library for
+writing a multi-dimensional performance space consisting of the dimensions (i)
+performance metric, (ii) call path, and (iii) system resource."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.07'}
+toolchainopts = {'verbose': False, 'pic': True}
+# verbose=False avoids the configure step to fail (-craype-verbose error)
+source_urls = ['http://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = [ 'e9beb140719c2ad3d971e1efb99e0916' ]
+# modextrapaths = { 'LD_LIBRARY_PATH': '../../../../../../../../../../../../../../../../../../usr/lib64' } # prevents conflict with DDT's Qt
```

Comment:
Small comment: is the above commented line needed?

Reply:
just a reminder about conflict with ddt, i will remove it next time.